JetUtilities/src/ClosureMaker.cc

```diff
@@ -385,10 +386,10 @@ void ClosureMaker::loopOverBins(TH2F* hvar, unsigned int iVarBin) {
                 continue;
             }
         }
-        else if(h.back()->GetEntries()<=4 && h.back()->GetEntries()>1) {
-            hClosure.back()->SetBinContent(ibin+1,h.back()->GetMean());
-            hClosure.back()->SetBinError(ibin+1,h.back()->GetMeanError());
-        }
+        // else if(h.back()->GetEntries()<=4 && h.back()->GetEntries()>1) {
```

Comment:
Why remove this?

Reply:
Well yeah, that's the logic set up there. What do you want to have happen?
1. >statThreshold entries and things proceed as normal
2. The part you commented out. If there is less than statThreshold entries, but it's not empty, do you want some fallback behavior? That was what these lines were for. If you don't want that behavior don't just comment these lines out; remove them entirely.
3. else skip this point

Reply:
If it does not exceed the statThreshold, I think it'd be good to skip the point to show that the correction is performing well. So I'll skip this point